examples/pytorch/test_pytorch_examples.py

```diff
@@ -200,6 +201,7 @@ def test_run_mlm(self):
             result = get_results(tmp_dir)
             self.assertLess(result["perplexity"], 42)
 
+    @unittest.skip("This test is flaky on CI.")
```

Comment:
Having something like  `# TODO: fixme (zack)` would be great, but if we expect this to be fixed quickly, it's fine.

Reply:
Agreed - done!